CHANGELOG.md

```diff
@@ -1,3 +1,154 @@
+ * Onion messages received to a blinded path we generated are now authenticated
+   implicitly rather than explicitly in blinded path `Context`s (#3917, #4144).
+ * `OMNameResolver::expire_pending_resolution` has been added for those who
+   cannot or do not wish to call `new_best_block` regularly (#3900).
+ * `lightning-liquidity`'s LSPS1 client now supports BOLT 12 payments (#3649).
+ * `LengthReadable::read` has been renamed `read_from_fixed_length_buffer` and
+   is implemented for all `Readable` (#3579).
+ * `LengthReadable` is now required to read various objects which consume the
+   full available buffer (#3640).
+ * structs in `lightning-liquidity` were renamed to be globally unique (#3583).
+ * Renamed `SpendableOutputDescriptor::outpoint` to `spendable_outpoint` (#3634)
+
+## Performance Improvements
+ * `ChainMonitor::load_existing_monitor` was added and should be used on startup
+   to load existing `ChannelMonitor`s rather than via `Persist`, avoiding
+   re-persisting each `ChannelMonitor` during startup (#3996).
+ * RGS data application was further sped up (#3581).
+
+## Bug Fixes
+ * `FilesystemStore::list` is now more robust against race conditions with
+   simultaneous `write`/`archive` operations (#3799).
+ * Pending async persistence of `ChannelMonitorUpdate`s required to forward an
+   HTLC can no longer result in the HTLC being forgotten if the channel is
+   force-closed (#3989).
+ * `lightning-liquidity`'s service support now properly responds to the
+   `ListProtocols` message (#3785).
+ * The fields in `SocketAddress::OnionV3` are now corectly parsed, and the
+   `Display` for such addresses is now lowercase (#4090).
+ * `PeerManager` is now more conservative about disconnecting peers which aren't
+   responding to pings in a timely manner. This may reduce disconnections
+   marginally when forwarding gossip to a slow peer (#4093, #4096).
+ * `BlindedMessagePath::new_with_dummy_hops` was added (but is not used by
+   default, #3726).
+ * Blinded path serialization is now padded to better hide its contents (#3177).
+ * In cases of incredibly long async monitor update or async signing operations,
+   LDK may have previously spuriously disconnected peers (#3721).
+ * Total dust exposure on a commitment now rounds correctly (#3572).
+
+## Backwards Compatibility
+ * `ChannelMonitor`s which were created prior to LDK 0.0.110 and which saw no
+   updates since LDK 0.0.116 may now fail to deserialize (#3638, #4146).
+ * After upgrading to 0.2, downgrading to versions of LDK prior to 0.0.123 is no
+   longer supported (#2933).
+ * Blinded paths generated by previous versions of LDK, except those generated
+   for inclusion in `Bolt12Offer`s will no longer be accepted (#3917).
+ * Once a channel has been spliced, LDK can no longer be downgraded.
+   `UserConfig::reject_inbound_splices` can be set to block inbound ones (#4150)
+ * LDK now requires the `channel_type` feature in line with spec updates (#3896)
+
+XXX release stats
```

Comment:
Reminder to update. Maybe XXX is part of the process?

Reply:
Yea, will do it once we get the other PRs landed.